Readme.md

```diff
@@ -30,16 +29,10 @@ If you're interested in only installing the `RxSwift` extensions, without the `R
 pod 'RxSwiftExt/Core'
 ```
 
-Using Swift 3:
-
-```ruby
-pod 'RxSwiftExt', '2.5.1'
-```
-
-If you use Swift 2.x:
+Using Swift 4:
 
 ```ruby
-pod 'RxSwiftExt', '1.2'
+pod 'RxSwiftExt', '~> 3'
```

Comment:
3.0 here?

Reply:
I don't think it matters that much (I also didn't modify the original 5.0 suggestion above). Many modules depend on the major (so 5.0 and 5.1 both work for a module).